### PR TITLE
Serve options requests with cors headers

### PR DESCRIPTION
**Pull request by q42jaap**
Somehow this was needed to get things running again.

### CODE REVIEW

File: src/server.js [hunk 1]
```diff
@@ -73,6 +76,13 @@ module.exports = class Server {
       });
   }
 
+  optionsHandler(req, res) {
+    res.set('Access-Control-Allow-Origin', req.get('origin'));
+    res.set('Access-Control-Allow-Credentials', true);
+
+    res.sendStatus(204);
```
Comment (JasperH8g) [inline src/server.js:83]: I don't get it.. how is this different than what we're already doing inside of `getHandler`? Only the status right? Why is this necessary?

Comment (q42jaap) [inline src/server.js:83]: See https://www.npmjs.com/package/cors#enabling-cors-pre-flight
The options request is needed for preflight requests, for GET request a preflight request is done when there are extra, non common headers, or when cookies are required to be sent (withCredentials).

**Comment by q42jaap** [comment]
#17 is a better implementation using the `cors` package.
